docs/uax-with-aj1/Request-to-revise-UAX-50-for-the-harmonization-with-Adobe-Japan1.md

```diff
@@ -20,8 +25,9 @@ Although the technical content of UAX 50 has been fairly stable since 2015, neit
 
 CITPC would like to request the Unicode Consortium to revisit the categorization of four characters in UAX 50. Although the current categorization is sensible, its practical benefits are less significant than the adoption cost, as we see it.  If these changes are accepted, existing Japanese AJ1 fonts will become compliant with UAX 50. 

```

Comment:
"CITPC would like to request the Unicode Consortium to revisit" -> "CITPC requests that the Unicode Consortium  revisit"